pkg/datastore/datastore.go

```diff
@@ -45,11 +45,18 @@ type Reader interface {
 // Writer is an interface that is used to manage the underlying datastore
 // for operator manifest.
 type Writer interface {
-	// GetPackageIDs returns a comma separated list of operator ID(s). Each ID
+	// GetPackageIDs returns a comma separated list of operator ID(s). This list
+	// includes operator(s) across all OperatorSource object(s). Each ID
 	// returned can be used to retrieve the manifest associated with the
 	// operator from underlying datastore.
 	GetPackageIDs() string
```

Comment:
why not make a list of strings instead?

Reply:
Good question, right now `CatalogSourceConfig.Spec.packages` is a string type. We need to change it to `[]string`. Could be a candidate for 4.1, I believe.